data/hai/hai outfits.txt

```diff
@@ -11,6 +11,38 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Phantom Pallet"
+	category "Systems"
+	cost 12000
+	thumbnail "outfit/phantom pallet"
+	"mass" 0.2
+	"unplunderable" 1
+	"energy consumption" 0.016667
+	"heat generation" 0.03334
+
+	# Replacing 30% of your cargo bay with these triples your cargo scan time.
+	"cargo space" -1
+	"cargo scan opacity" 15
+
+	description "This borderline-illegal smuggling tool is an Unfettered modification of the standard Hai Holopad. It is durable enough to survive a firefight and is hard to detect."
+	description "Holopads are small versions of the Hai Holovid Zone. They have holographic projectors, textured force fields, and phase array speakers. As a mid-priced but powerful option, it's often seen in meeting rooms, arcades, and personal gaming setups. Several can be packed together to make a larger holographic space."
+	description "Unfettered Phantom Pallets can hide under floors, in walls, or in ceilings while remaining undetected. Install them in your cargo bay and project fake cargo as sensor ghosts. This confuses scanners into scanning empty space instead of contraband. That can make the difference between seized cargo and delivering needed supplies."
+
+outfit "Holovid Zone"
+	category "Systems"
+	cost 320000
+	thumbnail "outfit/holovid zone"
+	"mass" 20
+	"outfit space" -20
+	"unplunderable" 1
+	"holographic entertainment" 1
+	"energy consumption" 0.5
+	"heat generation" 1.0
+	"cargo scan opacity" 150
+	"outfit scan opacity" 150
+	description "Behold the ultimate in immersive holographic entertainment: the Holovid Zone. Its holographic projectors and textured force fields present lifelike visuals. A phase array speaker system can emanate sounds from anywhere in the room. Connect this to your scanners to view a tactical situation as it evolves, or to your communications systems for a real-time 3D conference. Dazzle your passengers with the latest in holovids and games."
+	description "The possibilities are endless... especially for a sly captain who misuses the device. Should authorities seek contraband or illegal outfits, a Holovid Zone may fool them long enough for you to escape. Deathsticks are hard to find in a ship with fifty tons of holographic straw."
```

Comment:
```suggestion
	description "Hai holographic technology far exceeds the comparable human equivalents, and nowhere is this best exemplified than in the Holovid Zone. Its holographic projectors and textured force fields present lifelike visuals, while phase array speakers emanate sounds from anywhere in the room."
	description "	For the Hai, such technology is commonplace, being used to create 3D conference rooms, holovid theaters, immersive arcades, and more. Some Hai captains even install a Holovid Zone on their command deck to see a real-time tactical evaluation of a battle."
	description "	The possibilities are endless... especially for a sly captain who misuses the device. Should authorities seek contraband or illegal outfits, a Holovid Zone may fool them long enough for a smuggler to escape. Deathsticks are hard to find in a ship with fifty tons of holographic straw."
```

Reply:
This part doesn't make sense, given that Hai don't have battles:

> Some Hai captains even install a Holovid Zone on their command deck to see a real-time tactical evaluation of a battle.

Reply:
I mean, there is Wah Ki, isn't there?

Reply:
The Fettered Hai don't consider that a battle. And we know they don't install it in their ships for combat situations; none of the combat designs have them.

Reply:
Perhaps they did install them in the past.

Reply:
```suggestion
	description "Hai holographic technology far exceeds the comparable human equivalents, and nowhere is this best exemplified than in the Holovid Zone. Its holographic projectors and textured force fields present lifelike visuals, while phase array speakers emanate sounds from anywhere in the room."
	description "	For the Hai, such technology is commonplace, being used to create 3D conference rooms, holovid theaters, gaming arcades, and more. In the past, Hai captains used to install Holovid Zones on their command deck, allowing them to generate tactical evaluations of battles in real-time."
	description "	The possibilities are endless... especially for a sly captain who misuses the device. Should authorities seek contraband or illegal outfits, a Holovid Zone may fool them long enough for a smuggler to escape. Deathsticks are hard to find in a ship with fifty tons of holographic straw."
```